lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
```

Comment:
I would run those in series. If the `npm install` fails, don't do the other steps. This also allows you to output what exactly you have done, so that the developer knows (see attached file). 
![__ _fish](https://cloud.githubusercontent.com/assets/268934/6794552/5e8c2ec6-d190-11e4-9eba-fe878594c6ca.png)